db/schema/README.md

```diff
@@ -0,0 +1,35 @@
+## Schema
+
+This project uses the [mysql-patcher]() project to perform it's database schema migrations, however this is only used
+in development, testing and staging but not in production. Production is performed manually making sure each step
+succeeds properly.
+
+When adding a new patch file you need to provide two things:
+
+1. the forward and reverse patches in `db/schema/patch-xxx-xxx.sql`
+2. update `db/patch.js` to point to the new patches
+
+Each forward patch should perform it's actions and finally update the `dbMetadata` to the new patch level.
+
+Every reverse patch should perform the same actions but in exactly the same reverse order, but still update
+the `dbMetadata` table last. Also note that for this project we decided to comment out the reverse patch
+so that it couldn't be run accidentally.
+
+## Stored Procedures and Future Patches
+
+When a new stored procedure is added no further actions need to be taken. However, when a new version of a stored
+procedure is added, the older version should also be removed. However, due to the way we may have two subsequent
+versions of a stack accessing the database the old stored procedure should be removed in the next patch. As an example
+consider the following:
+
+* a stored procedure `doSomething_2` already exists
+* a new version `doSomething_3` is added in patch 12 during release 50
+* upon release:
+    * the old stack (release 49) is still using `doSomething_2`
+    * the new stack (release 50) is using `doSomething_3`
+* a new patch 13 should be added for release during release 51 which removes `doSomething_2`
+
+This works because by the time we get to release 51, the stack for release 49 has already been pulled down and nothing
+is using `doSomething_2` any longer.
+
+(Ends)
```

Comment:
Thanks @chilts.  Please also document the constent-table-ordering thing that's being introduced in this PR.